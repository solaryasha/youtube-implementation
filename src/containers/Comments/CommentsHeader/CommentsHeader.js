import React from 'react';
import './CommentsHeader.scss';
import { Button, Icon } from 'semantic-ui-react';


const CommentsHeader = (props) => {
    return (
        <div className='comments-header'>
            <h4>{props.commentsAmount} Comments</h4>
            <Button basic icon labelPosition='left'>
               <Icon name='align left'/>
               Sort By
            </Button>
        </div>
    );
}

export default CommentsHeader;