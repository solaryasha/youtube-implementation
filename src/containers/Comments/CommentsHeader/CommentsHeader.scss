.comments-header {
    h4 {
        display: inline-block;
        margin-right: 32px;
    }
}