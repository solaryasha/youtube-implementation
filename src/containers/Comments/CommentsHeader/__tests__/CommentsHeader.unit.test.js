import React from 'react';
import { shallow } from 'enzyme';
import CommentsHeader from '../CommentsHeader';

describe('CommentsHeader', () => {
    test('renders without props', () => {
        const wrapper = shallow(<CommentsHeader />);

        expect(wrapper).toMatchSnapshot();
    })

    test('renders without props', () => {
        const wrapper = shallow(<CommentsHeader commentsAmount={505050}/>);

        expect(wrapper).toMatchSnapshot();
    })
})