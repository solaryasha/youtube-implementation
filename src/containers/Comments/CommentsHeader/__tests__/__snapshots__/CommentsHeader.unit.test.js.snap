// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentsHeader renders without props 1`] = `
<div
  className="comments-header"
>
  <h4>
     Comments
  </h4>
  <Button
    as="button"
    basic={true}
    icon={true}
    labelPosition="left"
  >
    <Icon
      as="i"
      name="align left"
    />
    Sort By
  </Button>
</div>
`;

exports[`CommentsHeader renders without props 2`] = `
<div
  className="comments-header"
>
  <h4>
    505050
     Comments
  </h4>
  <Button
    as="button"
    basic={true}
    icon={true}
    labelPosition="left"
  >
    <Icon
      as="i"
      name="align left"
    />
    Sort By
  </Button>
</div>
`;
